src/argus/incident/validators.py

```diff
@@ -29,7 +29,8 @@ def validate_key(value):
 key_value_re = _lazy_re_compile(r"^[a-z0-9_]+=.+?\Z")
 key_value_validator = RegexValidator(
     key_value_re,
-    message='Enter a valid key-value construct: "key=value", where key and value are strings.',
+    message=f'Enter a valid key-value construct: "key=value", where key and value are strings.'
+            f'The key must consist of lowercase letters, numbers and underscores.',
```

Comment:
```suggestion
    message=f'Enter a valid key-value construct: "key=value", where key and value are strings.'
            f'The key can consist of lowercase letters, numbers and underscores.',
```

Otherwise it sounds like it must contain all of these